src/commcare_cloud/terraform/staging/terraform.tf

```diff
@@ -27,57 +26,57 @@ module "commcarehq" {
     {
       server_name           = "web0-staging"
       server_instance_type  = "t2.micro"
-      subnet_index          = 0
+      subnet                = "private-a"
     },
     {
       server_name           = "web1-staging"
       server_instance_type  = "t2.micro"
-      subnet_index          = 1
+      subnet                = "private-b"
     },
     {
       server_name           = "celery0-staging"
       server_instance_type  = "t2.large"
-      subnet_index          = 1
+      subnet                = "private-b"
```

Comment:
a subnet can not span multiple availability zones. That means `celery0-staging` is also in a different availability zone than machines in `private-a` and `private-c`. 
Maybe there is something I am missing here but, I think this and so do `pillow0-staging` should be in the same availability zone as other machines?

Reply:
I always thought AZ as a datacenter. They do provide very less latency (connected by low latency links ) but latency is even less if VMs are in the same AZ (datacenter). 
as you mentioned in another comment, AZs main purpose is to provide high availability so that if AZ1 fails we can shift the load to the machines in AZ2.